ansible/roles/test/tasks/lag_2.yml

```diff
@@ -72,12 +72,12 @@
 - name: test each lag interface minimum links and rate
   include: single_lag_test.yml
   with_items: lag_facts.names
-  when: lag_facts.lags[item]['po_config']['runner']['min_ports'] is defined
+  when: test_minlink|bool == true
```

Comment:
`lag_facts.lags[item]['po_config']['runner']['min_ports'] is defined and test_minlink|bool == true`